indexer/src/types/transaction.ts

```diff
@@ -400,6 +397,22 @@ function addSignature(
         r,
         s,
       });
+      const txHash = "0x" + Array.from(legacyTx.hash())
+        .map((b) => b.toString(16).padStart(2, "0"))
+        .join("");
+
+      if (
+        v < 35 &&
+        !Deno?.env.get("WHITE_LISTED_EIP_155_TRANSACTION_HASHES")?.includes(
```

Comment:
I think that if we use such an environment variable, we should configure it:
- In the indexer specific file `.env.example`
- In our docker files under `indexer` similar to other indexer environment variables (for example `DEFAULT_BLOCK_GAS_LIMIT`)

I say this because our current environment variable `WHITE_LISTED_EIP_155_TRANSACTION_HASHES` is configured for the Rust rpc part but not for the Typescript indexer.

Reply:
You're right, fixed. Thanks